src/loader/book.rs

```diff
@@ -0,0 +1,322 @@
+#![allow(missing_docs, unused_variables, unused_imports, dead_code)]
```

Comment:
Oops! I added those during development to decrease the signal-to-noise ratio of the compiler output. Looks like I forgot to remove it afterwards.